templates/registration/password_reset_form.html

```diff
@@ -35,16 +37,29 @@
     .h3{
         text-align: center;
     }
+    .fas {
+        font-size: 7em;
+        color: #ff8c00;
+        margin-bottom: 0.15em;
+    }
+    .btn-primary{
```

Comment:
Instead of changing property of `btn-primary` class, create a `submit-btn` class and write properties in that.

Also, remove the `btn-primary` CSS class from that button.